build/webTestReporter.js

```diff
@@ -6,11 +6,17 @@ const path = require('path');
 const { createServer } = require('http');
 const jsonc = require('jsonc-parser');
 const mocha = require('mocha');
+const dedent = require('dedent');
 const { EventEmitter } = require('events');
+const colors = require('colors');
+const core = require('@actions/core');
+const hashjs = require('hash.js');
+const glob = require('glob');
+const { ExtensionRootDir } = require('./constants');
 
 const settingsFile = path.join(__dirname, '..', 'src', 'test', 'datascience', '.vscode', 'settings.json');
-const webTestSummaryJsonFile = path.join(__dirname, '..', 'webtest.json');
-const webTestSummaryFile = path.join(__dirname, '..', 'webtest.txt');
+const webTestSummaryJsonFile = path.join(__dirname, '..', 'testresults.json');
```

Comment:
new name, all tests will now generate this file (not just web tests).
I.e. even regular (desktop) tests will generate this file, so we can dump the test results in a separate job as we did in web test, thus producing a cleaner, simpler  & consistent output for all tests.